lib/Controller/RoomController.php

```diff
@@ -1892,11 +1892,11 @@ public function verifyDialInPin(string $pin): DataResponse {
 	 * @param array{actorId?: string, actorType?: string, attendeeId?: int} $options Additional details to verify the validity of the request
 	 * @return DataResponse<Http::STATUS_OK, TalkRoom, array{}>|DataResponse<Http::STATUS_BAD_REQUEST|Http::STATUS_UNAUTHORIZED|Http::STATUS_NOT_FOUND|Http::STATUS_NOT_IMPLEMENTED, null, array{}>
 	 *
-	 *  200: Participant created successfully
```

Comment:
this is so annoying :see_no_evil: PHPStorm is double spacing when copy pasting :P